limitador/src/storage/redis/redis_cached.rs

```diff
@@ -292,43 +292,48 @@ impl CachedRedisStorageBuilder {
 async fn update_counters<C: ConnectionLike>(
     redis_conn: &mut C,
     counters_and_deltas: HashMap<Counter, Arc<CachedCounterValue>>,
-) -> Result<Vec<(Counter, i64, i64)>, StorageErr> {
+) -> Result<Vec<(Counter, i64, i64, i64)>, StorageErr> {
```

Comment:
the new value corresponds to the delta in the last position? maybe we could add an doc annotation to this fn

Reply:
I'll burn the whole ttl stuff, use absolute timestamps instead and then refactor this... 
I hate tuples... of 4 elements even more so.